jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EjbSchedulerService.java

```diff
@@ -146,30 +139,17 @@ private TimerMappingInfo getTimerMappinInfo(long processInstanceId, long timerId
     }
 
     private TimerMappingInfo getTimerMappingInfo(Function<EntityManager, List<TimerMappingInfo>> func) {
-        InternalRuntimeManager manager = ((GlobalTimerService) globalTimerService).getRuntimeManager();
```

Comment:
When using CMT, I do not think we need to create a differen transaction here anymonre